### ISSUE TITLE
`NcActions` doesn't work inside an element with a focus trap in a general case

### ISSUE THREAD

**Issue by ShGKme** [nextcloud-libraries/nextcloud-vue#4910]
`NcActions` handles `keydown` on the menu element. It works only when a menu item has a focus.

Because `NcActions` renders in the `body` by default, when it's used in a component with a focus trap, the focus trap blocks the focus and breaks keyboard navigation.

In a specific case, it can be adjusted by setting `container`. But not in a general case.

The current idea — provide a container for mounting from an element with a focus trap (overlapping elements like Navigation, Sidebar, Modal), and not to mount in the body by default.

**Comment by JuliaKirschenheuter** [comment]
Example:

![Screenshot from 2024-01-10 10-11-42](https://github.com/nextcloud-libraries/nextcloud-vue/assets/6078378/db88345b-9238-4068-8859-a9f186a34eae)

**Comment by szaimen** [comment]
I tested this on stable28 and focus trap does not seem to work:

https://github.com/nextcloud-libraries/nextcloud-vue/assets/42591237/e0a77487-b6f8-4e83-ba7b-3d9da8956e6b

**Comment by JuliaKirschenheuter** [comment]
Ok, server on stable28 still have an old `"@nextcloud/vue": "^8.4.0"` but needs v8.6.0. Moving to Parked.
